### PR TITLE
Update to latest Quests

### PR DESCRIPTION
**Pull request by PikaMug**
~~Also added Spigot repo to pom as spigot-api was not present elsewhere~~

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,6 +19,10 @@
     </properties>
 
     <repositories>
+        <repository>
+            <id>spigot</id>
+            <url>https://hub.spigotmc.org/nexus/content/repositories/snapshots/</url>
+        </repository>
```
Comment (mcmonkey4eva) [inline pom.xml:25]: This was intentionally excluded, and should not be added in an unrelated PR

File: src/main/java/com/denizenscript/depenizen/bukkit/commands/quests/QuestsCommand.java [hunk 2]
```diff
@@ -116,84 +115,73 @@ public void execute(ScriptEntry scriptEntry) {
...
-                    for (Quest quest : quests.getQuests()) {
+                    for (Quest quest : quests.getLoadedQuests()) {
                         if (quest.getId().equals(questId.asString())) {
                             quests.getQuester(player.getPlayerEntity().getUniqueId()).takeQuest(quest, override_checks.asBoolean());
                             break;
                         }
                     }
-                }
-                else if (points != null && points.asInt() > 0) {
+                } else if (points != null && points.asInt() > 0) {
```
Comment (mcmonkey4eva) [inline src/main/java/com/denizenscript/depenizen/bukkit/commands/quests/QuestsCommand.java:126]: This formatting should not have been changed

File: pom.xml (hunk 1 above)
Comment (PikaMug) [reply pom.xml:25]: Surprising as the project will not compile in a new environment without it, but I'll exclude it

File: src/main/java/com/denizenscript/depenizen/bukkit/commands/quests/QuestsCommand.java (hunk 2 above)
Comment (PikaMug) [reply src/main/java/com/denizenscript/depenizen/bukkit/commands/quests/QuestsCommand.java:126]: Not sure how that happened, will fix

**Pull request merged**